### ISSUE TITLE
Zest - Replace variables when running Action Invoke

### ISSUE THREAD

**Issue by thc202** [zaproxy/zaproxy#3511]
From OWASP ZAP User Group: https://groups.google.com/d/topic/zaproxy-users/DpSEmlvsYmI/discussion

Steps to reproduce the issue:
1. Run ZAP with Zest add-on installed;
2. Create a Stand Alone Zest script, that invokes another script passing a variable:
```JSON
{
  "about": "",
  "zestVersion": "0.8",
  "title": "Zest",
  "description": "",
  "prefix": "",
  "type": "StandAlone",
  "parameters": {
    "tokenStart": "{{",
    "tokenEnd": "}}",
    "tokens": {},
    "elementType": "ZestVariables"
  },
  "statements": [
    {
      "string": "Value",
      "variableName": "A",
      "index": 1,
      "enabled": true,
      "elementType": "ZestAssignString"
    },
    {
      "variableName": "C",
      "script": "/home/zap/scripts/scripts/standalone/ShowVar.js",
      "parameters": [
        [
          "A",
          "{{A}}"
        ]
      ],
      "index": 2,
      "enabled": true,
      "elementType": "ZestActionInvoke"
    }
  ],
  "authentication": [],
  "index": 0,
  "enabled": true,
  "elementType": "ZestScript"
}
```
3. Create another Stand Alone script (`ShowVar.js`), that shows the value of the variable passed in:
```JavaScript
org.parosproxy.paros.view.View.getSingleton().showMessageDialog(A)
```
4. Run the Zest script and note that the value is shown as `{{A}}` instead of the actual value of the variable `Value`.

Remarks:
This will require changing the Zest library.

ZAP version:
Version 2.6.0

Add-on:
"Zest - Graphical Security Scripting Language" version 24.

**Comment by thc202** [comment]
This is being addressed in mozilla/zest#97.

**Comment by thc202** [comment]
Fixed in zaproxy/zap-extensions#985.
